vignettes/my-vignette.Rmd

```diff
@@ -0,0 +1,216 @@
+---
+title: "safetyGraphics Shiny App - User Guide"
+author: "Jeremy Wildfire"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
```

Comment:
"Vignette Title" is where we can specify a name for this vignette.  It's what will appear on the CRAN page linking to the vignette.